learnify/mobile/lib/features/auth/forget-password/view/components/email_form.dart

```diff
@@ -0,0 +1,23 @@
+part of '../forget_password_screen.dart';
+
+class _EmailForm extends StatelessWidget {
+  const _EmailForm({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final ForgetPasswordViewModel model =
+        context.read<ForgetPasswordViewModel>();
+    return Form(
+        child: CustomTextFormField(
```

Comment:
You forgot the form key here, it isn't validating on button press. Please do tests on emulator.

Reply:
When I tried on my physical device, it was differentiating between valid and not valid emails. Maybe we should discuss in more detail

Reply:
If you look at the code, you can see that `Form` widget doesn't have the `key` parameter. However, it was appearing in your previous implementation. You mistakenly deleted it. You can your previous implementation:
https://github.com/bounswe/bounswe2022group2/pull/382/commits/5973e4707ced016b6887efacbe6ca20d2c4b64c4#diff-b04afa14bdf369cdebb9e860723b952873db16d28da508aaf2e0e14679c9aa0c